apps/discovery_api/test/unit/discovery_api_web/controllers/tableau_controller_table_info_test.exs

```diff
@@ -20,8 +20,8 @@ defmodule DiscoveryApiWeb.TableauControllerTableInfoTest do
       ]
 
       allow(Model.get_all(), return: mock_dataset_summaries, meck_options: [:passthrough])
-      allow(ModelAccessUtils.has_access?(%{id: "private"}, any()), return: false)
       allow(ModelAccessUtils.has_access?(any(), any()), return: true)
+      allow(ModelAccessUtils.has_access?(%{id: "private"}, any()), return: false)
```

Comment:
😲

Reply:
Yea in new placebo the clause order is reversed.  Now the most generic clause goes first so you can elevate it to the setup block.  I  did not perform that level of refactoring though, i merely made things work in their current place.